pkg/util/clusteragent/clusteragent.go

```diff
@@ -36,15 +39,22 @@ Client to query the Datadog Cluster Agent (DCA) API.
 const (
 	authorizationHeaderKey = "Authorization"
 	// RealIPHeader refers to the cluster level check runner ip passed in the request headers
-	RealIPHeader = "X-Real-Ip"
+	RealIPHeader          = "X-Real-Ip"
+	languageDetectionPath = "api/v1/languagedetection"
 )
 
 var globalClusterAgentClient *DCAClient
 
 type metadataNames []string
 
+// LanguageDetectionClient defines the method to send a message to the Cluster-Agent
+type LanguageDetectionClient interface {
```

Comment:
I'm not sure the definition of this interface belongs to this package, perhaps it should be named more clearly in the `languagedetection` package.

Reply:
Indeed but then it creates a circular dependency. It's also the current pattern that we follow for other components

Reply:
You don't have to embed the interface in this one. You define the interface in `languagedetection` and it happens that DCAClientInterface implements it.

Reply:
But then we need to define the methods in two different places ?